day2/src/index.js

```diff
@@ -0,0 +1,24 @@
+import './styles/main.scss';
+import hide from './modules/switch.js'
+import loadProducts from './modules/productHTML.js';
+import {products, onCartTrue } from './modules/products.js';
+import loadCart from './modules/cartHTML.js';
+import { cart, updateCart } from './modules/cart.js';
+
+if(localStorage.local) {
+  let parsedLocal = JSON.parse(localStorage.local);
+  parsedLocal.forEach((each) => {
+    if (each.onCart) {
+      onCartTrue(each.id, products);
+    }
+    else {
+      document.querySelector('.card-container').innerHTML =  `<span class="message">The cart is empty! :(</span>`;
```

Comment:
- [ ] Make sure it is working and is in the center of the page-like layout.
_*Hint you use `min-height: 50vh` or whatever you prefer_